delivery_ups_oca/models/ups_request.py

```diff
@@ -46,10 +48,13 @@ def _process_reply(
         self.carrier.log_xml(status or "", "ups_last_response")
         errors = status.get("response", {}).get("errors")
         if errors:
-            _logger.info(
-                _("Sending to UPS: %s")
-                % ("\n".join("%(code)s %(message)s" % error for error in errors),)
+            msg = _("Sending to UPS: %s") % (
+                "\n".join("%(code)s %(message)s" % error for error in errors),
             )
+            if self.skip_errors:
```

Comment:
Why not passing this by argument?

Reply:
I wanted to avoid that all the calls have to add the parameter that I think should be at a general level.

Reply:
Well, this kind of transactional calls should be stateless.

Reply:
I have added the `_raise_for_status()` function for the errors issue. Do you think it's a good idea to do it like this?

Reply:
Well, sounds good. How is it done in other Odoo integrations?

Reply:
Each integration does it in a different way, it would be convenient to establish a standard to manage errors in the same way in all integrations.

Reply:
Look if Odoo SA has set any standard, and let's follow it.